packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -22,6 +22,9 @@ export function CrudGenerator({
 export interface CrudSingleGeneratorOptions {
     targetDirectory: string;
     requiredPermission?: string[] | string;
+    create?: boolean;
+    update?: boolean;
+    delete?: boolean;
```

Comment:
Single Generator doesn't generate create or delete, please remove that.

Reply:
Ah, I see it does not affect the resolver.